db/postgresql/schema.sql

```diff
@@ -319,20 +333,55 @@ CREATE TABLE "sch_usr_notification_email"
 );
 CREATE INDEX "ix_SCH_USR_NOTIFICATION_EMAIL_email_id" ON "sch_usr_notification_email" ("email_id");
 
+CREATE SEQUENCE "seq_organization" START 1;
+CREATE TABLE "organization"
+(
+  "id" INTEGER,
+  "name" varchar(256) NOT NULL,
+  PRIMARY KEY ("id"),
+  UNIQUE ("name")
+);
+CREATE INDEX "ix_organization_name" ON "organization" ("name");
+
 CREATE SEQUENCE "seq_usr_user" START 1;
 CREATE TABLE "usr_user"
 (
   "id" INTEGER,
-  "username" VARCHAR(25) NOT NULL,
+  "username" VARCHAR(256) NOT NULL,
   "email_id" INTEGER,
-  "u_password" CHAR(142) NOT NULL,
-  "full_name" VARCHAR(55),
+  "u_password" CHAR(256) NOT NULL,
+  "full_name" VARCHAR(256),
   "table_rows" INTEGER,
   PRIMARY KEY ("id"),
   UNIQUE ("username")
 );
 CREATE INDEX "ix_USR_USER_email_id" ON "usr_user" ("email_id");
 
+CREATE SEQUENCE "seq_permission" START WITH 1;
+CREATE TABLE "permission" (
+  "id" INTEGER,
+  "name" varchar(142) NOT NULL,
+  "rwonly" boolean,
```

Comment:
** Solved: OK**
The motivation is that certain permissions are applied depending on the visibility of the object (pipeline). So the idea of the column is to express whether the permission should apply anytime or only if the visibility of the shared object is RW.